### DISCUSSION TITLE
Dates not imported from 1Password correctly.

### DISCUSSION THREAD

**Discussion by EmnaX** [dani-garcia/vaultwarden#2093] [Q&A]
Hello,
I have recently moved over to Vaultwarden from 1Password. I exported my 1Password vaults as 1Password Interchange Format (.1pif) but when I import them into Vaultwarden, the date fields are shown as a long number instead of a date. For example:  

Date in 1Password: 2021-11-09
Date in Vaultwarden: 1636482973
  
Here are the versions for both:
1Password: 7.9.1 (70901007)
Vaultwarden: 2.23.0 (Bitwarden)

**Comment by BlackDex** [comment]
Processing of the imports are done by the client, in this case the web-vault. Vaultwarden only stores the information it gets, which is all encrypted. This would probably be best to report at Bitwarden. But i think that is already the case.
